osmaxx/profile/forms.py

```diff
@@ -1,12 +1,14 @@
 from crispy_forms.helper import FormHelper
 from crispy_forms.layout import Submit, Layout
-from django.contrib.auth import get_user_model
 from django.core.urlresolvers import reverse
 from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from osmaxx.profile.models import Profile
 
 
 class ProfileForm(forms.ModelForm):
-    email = forms.EmailField(max_length=200, required=True)
+    unverified_email = forms.EmailField(max_length=200, required=True, label=_('email'))
```

Comment:
Why, is there anything else than an address? Or is it not clear enough without the address?